### DISCUSSION TITLE
Bugs on version 1.17.0

### DISCUSSION THREAD

**Discussion by cyberdarvin** [mockoon/mockoon#646] [General]
I **think** there are 2 bugs on version 1.17.0 (if they are not bugs maybe I am not understanding the correct new workflow):

- when you try to import/export --> Mockoon's format --> **Import from clipboard**: instead of copy from clipboard, Mockoon opens file explorer to "**save**" your new enviroment.
This is a big issue, because if you don't read title of popup, you could think that your opening an environment, but you are going to overwrite the existing one.

- when you try to  import/export --> Mockoon's format --> **Import from file**: after selected a mockoon file, app opens a new file explorer to save an additional file.

**Comment by 255kb** [comment]
These are not bugs, but I admit it's a bit confusing. 
When you import (either from a file or the clipboard) you must choose a destination where to save your "new" environment. Exported environments, either in files or clipboard, were never designed to be opened as-is. 

However, this will change soon with v1.18.0. 
Import/export in Mockoon's format is now obsolete and has no advantage over sharing environment files as-is. Keeping this feature is really confusing, that is why the import/export options [will be removed](/issues/626) (of course, OpenAPI stays). The clipboard feature will stay but will copy the environment JSON description in the regular format, not as a wrapped exported like before. Though, adding a new environment from the clipboard will still require choosing a destination where to save it. 

I hope it's clearer! Let me know if you need more info.
